### DISCUSSION TITLE
Is it possible run it on a Cortex-M4 MCU?

### DISCUSSION THREAD

**Discussion by Femtometer** [SquirrelJME/SquirrelJME#102] [General Questions]
I'd like to make a mcu board that can run apps.

**Comment by XerTheSquirrel** [comment]
Hi!

What are the specifications for the CPU and board? What is the memory map like?

**Reply by Femtometer** [reply]
ARM Cortex-M4 1MB RAM and 4MB Flash, I've made my code run like an 'APP' base on threadX.

**Reply by XerTheSquirrel** [reply]
The RAM should be enough although the flash is a bit small, this would need about 32MiB or so.
